oaknational/Oak-Web-Application
# [CUR-285] fix: removed 'examboard' text from school phase picker

## Description

- School phase picker had "exam board" above the school phase.

## How to test

1. Go to https://deploy-preview-1950--oak-web-application.netlify.thenational.academy
2. Click on subject phase picker
3. You should see the heading for School Picker and no other text 

## Screenshots

How it used to look (delete if n/a):
<img width="1027" alt="Screenshot 2023-09-28 at 10 53 29" src="https://github.com/oaknational/Oak-Web-Application/assets/32605982/f2679f41-d0c5-4710-b198-c6075d645dc2">

How it should now look:
![Screenshot 2023-09-28 at 16 44 20](https://github.com/oaknational/Oak-Web-Application/assets/32605982/99e5bcdf-460e-4851-b8e4-dfce732f82bf)

## Checklist
